package.json

```diff
@@ -116,12 +117,16 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.1.5",
+    "autoprefixer": "^9.5.1",
+    "bootstrap-sass": "^3.4.1",
     "classnames": "^2.2.6",
     "date-arithmetic": "^3.0.0",
     "dom-helpers": "^3.4.0",
     "invariant": "^2.2.4",
     "lodash": "^4.17.11",
     "memoize-one": "^4.0.3",
+    "postcss": "^7.0.16",
+    "postcss-cli": "^6.1.2",
```

Comment:
these all belong in devDeps

Reply:
On it (still getting used to yarn again)

Reply:
Done